pkg/kv/db_test.go

```diff
@@ -288,15 +288,18 @@ func TestAuthentication(t *testing.T) {
 func TestTxnDelRangeIntentResolutionCounts(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	var intentResolutionCount int64
-	var resolveIntentCount int64
 	params := base.TestServerArgs{
 		Knobs: base.TestingKnobs{
 			Store: &storage.StoreTestingKnobs{
 				NumKeysEvaluatedForRangeIntentResolution: &intentResolutionCount,
 				TestingCommandFilter: func(filterArgs storagebase.FilterArgs) *roachpb.Error {
-					_, ok := filterArgs.Req.(*roachpb.ResolveIntentRequest)
+					req, ok := filterArgs.Req.(*roachpb.ResolveIntentRequest)
 					if ok {
-						atomic.AddInt64(&resolveIntentCount, 1)
+						key := req.Header().Key.String()
+						// cannot be an intent from the range being scanned.
```

Comment:
Capitalization, and also the comment is confusing. If the below condition holds, it *is* one of the range being scanned, no?